include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -169,40 +169,40 @@ typedef void(ORT_API_CALL* OrtLoggingFunction)(
 /**
  * \param out Should be freed by `OrtReleaseEnv` after use
  */
-ORT_API_STATUS(OrtCreateEnv, OrtLoggingLevel default_warning_level, _In_ const char* logid, _Out_ OrtEnv** out)
+ORT_API_STATUS(OrtCreateEnv, OrtLoggingLevel default_logging_level, _In_ const char* logid, _Outptr_ OrtEnv** out)
 ORT_ALL_ARGS_NONNULL;
 
 /**
  * \param out Should be freed by `OrtReleaseEnv` after use
  */
 ORT_API_STATUS(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_function,
-               _In_opt_ void* logger_param, OrtLoggingLevel default_warning_level,
+               _In_opt_ void* logger_param, OrtLoggingLevel default_logging_level,
                _In_ const char* logid,
-               _Out_ OrtEnv** out);
+               _Outptr_ OrtEnv** out);
```

Comment:
reference - https://docs.microsoft.com/en-us/visualstudio/code-quality/understanding-sal?view=vs-2019#example-the-_outptr_-annotation